### PR TITLE
Add instruments section in runcards

### PR DESCRIPTION
**Pull request by stavros11**
Updates runcards for the new layout implemented in qiboteam/qibolab#585. I would like to wait for qiboteam/qibolab#508 before merging this, because a few more changes may be required in the interface.

I also added a workflow that is running the platform creation test with qibolab main. We can use this to check if the runcards are compatible with main before releasing qibolab. @AleCandido can you help me on how to properly activate the poetry environment in that workflow
https://github.com/qiboteam/qibolab_platforms_qrc/blob/1d0e89d00ce159d5c9fad12bb500721113aa5047/.github/workflows/main.yml#L30

### CODE REVIEW

File: .github/workflows/main.yml
Comment (AleCandido) [inline .github/workflows/main.yml:32]:
```suggestion
          QIBOLAB_PLATFORMS=$(realpath .) poetry run pytest tests/
```

File: .github/workflows/main.yml
Comment (AleCandido) [inline .github/workflows/main.yml:27]: This is not required at the end of a block (the new shell will re-enter in the default directory).
```suggestion
```

File: iqm5q_q0.yml
Comment (andrea-pasquale) [inline iqm5q_q0.yml:16]: Is it possible to omit `power` instead of putting `null`?

**Review by andrea-pasquale — COMMENTED** [review]
Thanks @stavros11, LGTM.
Just a small comment.
As soon as CI passes I will approve

File: .github/workflows/main.yml
Comment (stavros11) [reply .github/workflows/main.yml:32]: This will not work as it is because we need to execute `poetry run` from within the qibolab project directory. Instead I tried 
```sh
cd qibolab
QIBOLAB_PLATFORMS=$(realpath .) poetry run pytest ../tests/
```
 here https://github.com/qiboteam/qibolab_platforms_qrc/actions/runs/6310932947/job/17133888194 but I get that `qibolab` is not installed so maybe the environment is not properly activated.

Comment (AleCandido) [reply .github/workflows/main.yml:32]: You're right about the change of directory, the problem with installation is not because of environment activation, but because `qibolab` has never been installed before because of the `--no-root` flag

https://python-poetry.org/docs/cli/

File: .github/workflows/main.yml
Comment (AleCandido) [inline .github/workflows/main.yml:26]:
```suggestion
          poetry install --no-interaction --with tests --all-extras
```

File: .github/workflows/main.yml
Comment (AleCandido) [reply .github/workflows/main.yml:32]: (`--no-root` means: *install all dependencies but not the package itself*)

Comment (stavros11) [reply .github/workflows/main.yml]: Thanks, I just blindly copied the other workflow without checking what the options were doing. It is working now.

File: iqm5q_q0.yml
Comment (stavros11) [reply iqm5q_q0.yml]: Seems possible because the LOs use `None` as default value anyway, thanks.

**Comment by stavros11** [comment]
> Thanks @stavros11, LGTM. Just a small comment. As soon as CI passes I will approve

Thank you for the reviews. The workflow using qibolab main is passing, the other one will require to release qibolab. But I think it makes sense to merge this only after releasing qibolab, in order to maintain the cluster environment.

**Review by andrea-pasquale — APPROVED** [review]

**Review by AleCandido — APPROVED** [review]

**Comment by stavros11** [comment]
Merging this because I need to make the main workflow available in the other open PRs. We can fix the couplers in a different PR after qiboteam/qibolab#508 is merged.

**Pull request merged**
